### PR TITLE
51 frontpage with signature

### PR DESCRIPTION
**Pull request by Kechr**

### CODE REVIEW

File: project_proposal/front_page.tex [hunk 1]
```diff
@@ -0,0 +1,47 @@
+\begin{titlepage}
+\begin{center}
+% Title
+{ \LARGE \bfseries Bierproductie \\[0.4cm]}
+A management system for brewing machines
+\begin{figure}[H]
+\centering 
+\includegraphics[scale=0.5]{figures/creditoro.png}
```
Comment (kkmp-dk) [inline project_proposal/front_page.tex]: Missing a frontpage figure.

File: project_proposal/signatures.tex [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+% VARIABLES
+\newcounter{PROD}
+\setcounter{PROD} {0}
+
+%%%%
+
+
+% Jakob Jakob Jakob Jakob Jakob Jakob Jakob Jakob Jakob Jakob Jakob
+\ifnum \value{PROD}=1
+    \includegraphics[scale=0.07]{signatures/signatureJR.jpg}
```
Comment (kkmp-dk) [inline project_proposal/signatures.tex:37]: We are also missing signatures files, or people need to make a new one again

**Review by kkmp-dk — CHANGES_REQUESTED** [review]

File: project_proposal/front_page.tex (hunk 1 above)
Comment (Kechr) [reply project_proposal/front_page.tex]: Yup, wasn't sure what figure to go with.

File: project_proposal/signatures.tex (hunk 2 above)
Comment (Kechr) [reply project_proposal/signatures.tex:37]: I think we still have them from last term, I'll check tomorrow.

File: project_proposal/front_page.tex (hunk 1 above)
Comment (kkmp-dk) [reply project_proposal/front_page.tex]: Maybe we could go with the one from project proposal made by @nymann

Comment (Kechr) [reply project_proposal/front_page.tex]: I've added the system drawing

File: project_proposal/signatures.tex (hunk 2 above)
Comment (Kechr) [reply project_proposal/signatures.tex:37]: I found the signatures and added them

**Pull request merged**